index.html

```diff
@@ -24,25 +24,25 @@ <h1 class="color-one title">sugustina.dev</h1>
       </div>
     </div>
   </header>
-  <section>
+  <section class="banner-color">
```

Comment:
Try to use classes that not only reference a single property, for some cases is good, but not probably in this case.